frontend/src/components/StartTestDialog/StartTestDialog.tsx

```diff
@@ -14,22 +14,20 @@ import {
 
```

Comment:
StartTestDialog should be in the Learning course folder, components folder contains only reusable (at different pages) components (Button, Image, etc.)

Reply:
yeah, this dialog  is used not only on learning page

Reply:
Ok, got it